_includes/get_a_quote.html

```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
         We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
     </p>
     <p>
-        When you hire us, we will have a trial period of a month, after which we will evaluate our working relationship, to ensure that you are happy with the work we are doing for you.
+        If you get in touch, we will first arrange a meeting to talk about your concrete business situation.
+        We will suggest a small project for you to get familiar with how we work, and then hopefully initiate a longer, sustainable relationship, where we help you improve your business's software.
```

Comment:
I would remove 'will' ->
'We suggest a small project..'
I think that 'and then hopefully' sound a bit like we don't believe in our self. Maybe we could expand it to something like:
'.. and if we're a good match then hopefully..' / 'if it's a right fit then hopefully..'